doc/src/install_guide/installation_guide.md

```diff
@@ -54,14 +40,16 @@ $ sudo make modules_install
 $ sudo make install
 ```
 
-When installed finished, reboot your system.
-When the system login again, check the kernel version is correctly or not.
+Build linux DFL Kernel instructions please also refer to: https://github.com/OPAE/linux-dfl/wiki/Build-the-linux-dfl-kernel
+
+When install finished, reboot your system.
+When the system login again, verify the kernel version is correct.
```

Comment:
Looks good.

One minor change. On line 48, can you add "For example" since this is just an example output?
When the system login again, verify the kernel version is correct.  For example:

Reply:
fixed